src/app/static/src/js/components/runReport/runReport.vue

```diff
@@ -53,23 +53,12 @@
                                 :params="parameterValues"></parameter-list>
             </div>
             <div v-if="showChangelog">
-                <div v-if="showChangeMessage" id="changelog-message" class="form-group row">
-                    <label for="changelogMessage" class="col-sm-2 col-form-label text-right">Changelog Message</label>
-                    <div class="col-sm-6">
-                        <textarea class="form-control" id="changelogMessage" v-model="changeLogMessageValue"
-                                  rows="2"></textarea>
-                    </div>
-                </div>
-                <div id="changelog-type" class="form-group row">
-                    <label for="changelogType" class="col-sm-2 col-form-label text-right">Changelog Type</label>
-                    <div class="col-sm-6">
-                        <select class="form-control" id="changelogType" v-model="changeLogTypeValue">
-                            <option v-for="option in metadata.changelog_types" :value="option">
-                                {{ option }}
-                            </option>
-                        </select>
-                    </div>
-                </div>
+            <change-log :show-change-message="showChangeMessage"
+                        :report-metadata="metadata"
+                        :changelog-style="changelogStyle"
+                        @changelogMessage="handleChangeLogMessageValue"
+                        @changelogType="handleChangeLogTypeValue">
+            </change-log>
```

Comment:
For proper separation of concerns I think that `<change-log/>` should accept one metadata prop: `changelog-types`. It doesn't need to know anything else about reports, metadata etc.